### ISSUE TITLE
No menu in libreoffice

### ISSUE THREAD

**Issue by nisitiiapi** [netblue30/firejail#2102]
firejail version 0.9.54
Linux Mint 19

![screenshot from 2018-08-30 22-30-29](https://user-images.githubusercontent.com/42857414/44893855-33e71480-acaa-11e8-8047-853e32e23976.png)

Libreoffice has no menu bar when run in firejail. Without firejail, the menu is present as normal. The problem is consistent across all my computers running Linux Mint 19 and firejail 0.9.54. I am using the default libreoffice profile from 0.9.54 except "apparmor" is uncommented since commenting it prevents libreoffice from starting.

I have tried commenting out everything in libreoffice.profile and nothing brings back the menu.  The only thing I cannot comment out is apparmor -- that prevents libreoffice from starting at all (as does firejail --noprofile libreoffice).

I am suspicious that this could be something with apparmor and firejail since that's the only thing I cannot really test. Running without apparmor in the profile or --noprofile just throws a fatal error of:
`[context="user"] caught unexpected com.sun.star.deployment.DeploymentException: Extension Manager: failed to read data entry in configuration backend db: file:///home/<user>/.config/libreoffice/4/user/uno_packages/cache/registry/com.sun.star.comp.deployment.configuration.PackageRegistryBackend/backenddb.xml`
Since I can't disable apparmor in the profile without libreoffice throwing that error and crashing, I can't seem to track it down for sure.

**Comment by Vincent43** [comment]
Check for libreoffice AppArmor profile with `sudo aa-status` then disable it with `sudo aa-disable <apparmor-profile-ame>`. You may  also look at https://github.com/netblue30/firejail/issues/1917

**Comment by nisitiiapi** [comment]
Thank you very much.  There are 4 apparmor profiles for libreoffice in Mint 19.  The offending profile seems to be usr.lib.libreoffice.program.soffice.bin, though I also had to disable usr.libreoffice.program.oosplash (caused a different error in firejail with the other apparmor profile disabled). I was able to keep usr.lib.libreoffice.program.senddoc and usr.lib.libreoffice.program.xpdfimport enforced.

In any event, disabling the two profiles resolved the problem and allowed me to disable apparmor in the libreoffice firejail profile without throwing and error and the menu is now back.

Thanks again.
